docs/en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -3,13 +3,13 @@ summary: Add versioning to your database content through the Versioned extension
 
 # Versioning
 
-Database content in SilverStripe can be "staged" before its publication, as well as track all changes through the 
+Database content in SilverStripe can be "staged" before its publication, as well as track all changes through the
```

Comment:
I'd be careful of trimming trailing whitespaces in markdown - as two spaces translate to a `<br>` in html

Reply:
Well we have no need for a `<br>` here so it should be ok.

Reply:
yeah, I don't want to spend time going through all the whitespace changes to see if you did end up trimming a double space at the end :P

it's pretty much "please don't trim for markdown"

Reply:
> I'd be careful of trimming trailing whitespaces in markdown - as two spaces translate to a <br> in html

TBH, if people want a break, they should be using a double new line not two spaces - MD should be readable in both it's raw form and it's rendered form.

Reply:
that's true, but two new lines means a new paragraph block right?